cli/cache_proxy/cache_proxy.go

```diff
@@ -123,7 +125,9 @@ func (p *CacheProxy) GetCapabilities(ctx context.Context, req *repb.GetCapabilit
 			log.Infof("Remote capabilities: %s", string(b))
 		}
```

Comment:
```suggestion
		}
		return res
```

Reply:
(or better yet, invert the logic so that this clause is checking `err != nil` and doing the fallback in that case - generally `err == nil` feels a bit harder to read than `err != nil` since the latter is so much more common)

Reply:
Good catch, inverted the logic and added the missing return. PTAL